### PR TITLE
Ucat faculties and administration coordination

### PR DESCRIPTION
**Pull request by fabiolayerim**

### CODE REVIEW

**Review by maacaro — APPROVED** [review]
LGTM

File: cypress/integration/admission-coordination.spec.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+describe("check list page for administration coordination testing", () => {
+  it("visits login and then administration coordination list page", () => {
+    cy.visit("/");
+    cy.get("[data-cy=page-login]").should("exist");
+    cy.get("input[data-cy=username]").should("exist").type("fabiolarueda");
+    cy.get("input[data-cy=password]").should("exist").type("ucat12345");
+    cy.get("[data-cy=send]").should("exist").click();
+
+    cy.visit("/admission-coordination");
```
Comment (hamalsolutions) [inline cypress/integration/admission-coordination.spec.js:9]: @fabiolayerim is there a link to click nad go to this page, or should be redirect after login?

File: src/pages/admissionCoordination/admission.js [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+                <tr>
+                  <th>Carrera</th>
+                  <th>Cédula</th>
+                  <th>Nombre</th>
+                  <th>Fecha Aprobado</th>
+                  <th>Referencia</th>
+                  <th>Nº de Expediente</th>
+                  <th>Observaciones</th>
+                  <th>Formalizar Inscripción</th>
+                  <th>Adjuntar Planilla</th>
```
Comment (hamalsolutions) [inline src/pages/admissionCoordination/admission.js:36]: @fabiolayerim why in the dashboard we need to "adjuntar planilla"

File: src/pages/admissionCoordination/admission.js [hunk 3]
```diff
@@ -0,0 +1,241 @@
+import React from "react";
```
Comment (hamalsolutions) [inline src/pages/admissionCoordination/admission.js:1]: @fabiolayerim this page is a list of `applicants` I would call this `pages/applicants/applicants`

File: src/pages/admissionCoordination/admissionContainer.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
```
Comment (hamalsolutions) [inline src/pages/admissionCoordination/admissionContainer.js:1]: @fabiolayerim I would call this `pages/applicants/applicantsContainer`
we migth use this later to build the other views of this list you now in admmisiones, socie-economico and so on and so forth

File: src/pages/admissionCoordination/admission.js (hunk 2 above)
Comment (hamalsolutions) [reply src/pages/admissionCoordination/admission.js:36]: @fabiolayerim shouldn't we just have la planilla and siplayed in a modal?

File: src/pages/faculties/faculties.js [hunk 5]
```diff
@@ -0,0 +1,241 @@
+import React from "react";
```
Comment (hamalsolutions) [inline src/pages/faculties/faculties.js:1]: @fabiolayerim can we just use the applicants componets with diferents props to create this view?

File: src/pages/admissionCoordination/admission.js [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+                </tr>
+              </tbody>
+            </Table>
+          </Col>
+        </Row>
+      </Container>
+    </div>
+  );
+};
+
```
Comment (hamalsolutions) [inline src/pages/admissionCoordination/admission.js:240]: @fabiolayerim why we dont have here CTA to aprove or not approve the check like there are in faculties

**Review by hamalsolutions — CHANGES_REQUESTED** [review]
@fabiolayerim creo que se pueden unir facultiescon admissiones y crear un solo componente Applicants o ApplicantsList 
en cada una de estas listas se debe poder observar los documentos y poder aprobar o no la diferncia es que son distintos ususarios

File: cypress/integration/admission-coordination.spec.js (hunk 1 above)
Comment (fabiolayerim) [reply cypress/integration/admission-coordination.spec.js:9]: currently the view is just jsx and has no redirect validation or a link

File: src/App.js [hunk 7]
```diff
@@ -16,7 +16,7 @@ function App() {
           <Route path="/faculties">
             <Faculties />
           </Route>
-          <Route path="/admission-coordination">
+          <Route path="/applicants">
             <Admission />
```
Comment (maacaro) [inline src/App.js:20]: @fabiolayerim shouldn't be applicants too?

**Review by maacaro — CHANGES_REQUESTED** [review]
@fabiolayerim lo que creo es que esas 2 vistas con un solo componente que va cargar datos dependiendo del ususuario lo podemos discutir en el meeting

**Review by maacaro — APPROVED** [review]
LGTM

**Review by hamalsolutions — APPROVED** [review]
LGTM

**Pull request merged**